descriptors/ModuleDescriptor-template.json

```diff
@@ -2,6 +2,35 @@
   "id": "${artifactId}-${version}",
   "name": "Circulation Module",
   "provides": [
+    {
+      "id": "requests-reports",
+      "version": "0.1",
```

Comment:
Not a big deal and certainly not anything to stop the merge, but "0.1" version suggests to me a pre-release. Would "1.0" be more appropriate here or is there truly more work to be done in this interface before it could be considered 1.0?

Reply:
This is a temporary interim solution for generating reports, I do not think that it needs a v1.0.
@marcjohnson-kint What do you think about it?

Reply:
@mreno-EBSCO I asked for this to be a pre-release version because it is only used in one situation and I didn't don't think the API is established.

I asked for a similar change for the batch APIs, which we know are unstable at this point.

It is mostly to avoid having to coordinate breaking changes early in development.

This was before the API switched to use the standard request representation.

@mreno-EBSCO  If you feel strongly about it, we can change it to 1.0?

Reply:
Absolutely not, which is why I said it should not block the merge. I just wanted clarification, which I now have.

Reply:
ok, then everything will remain as it is

Reply:
@mreno-EBSCO Sorry, bouncing around between things means I'm reading stuff in a hurry